aws/resource_aws_storagegateway_gateway_test.go

```diff
@@ -1078,14 +1127,14 @@ resource "aws_directory_service_directory" "test" {
   }
 }
 
-`, rName, domain)
+`, rName, domainName)
 }
 
-func testAccAWSStorageGatewayGatewayConfig_DirectoryServiceMicrosoftAD(rName, domain string) string {
+func testAccAWSStorageGatewayGatewayConfig_DirectoryServiceMicrosoftAD(rName, domainName string) string {
 	return fmt.Sprintf(`
 resource "aws_directory_service_directory" "test" {
   edition  = "Standard"
-  name     = %[2]q
+  name     = "terraformtesting.com"
```

Comment:
This should use the `domainName` parameter

```suggestion
  name     = %[2]q
```

Reply:
Confirmed, the AccTests caught that. Fixed.